packages/styles/src/atoms/icons.ts

```diff
@@ -6,50 +9,53 @@ type FontAwesomeIconSize = 'xs' | 'lg' | 'sm' | '1x' | '2x' | '3x' | '4x' | '5x'
 
 export type IconSize = 'small' | 'medium' | 'large'
 
-export type IconStatus = 'red' | 'green' | 'amber' | 'info' | 'default' | 'primary' | 'secondary'
+export type IconStatus = 'default' | 'red' | 'green' | 'amber' | 'info' | 'primary' | 'secondary' | 'link'
```

Comment:
I'm still not a fan of this 'status' attribute - why not just call it colour? What's the concept of an icon's "status"?